prover/src/trace/poly_table.rs

```diff
@@ -43,15 +43,16 @@ impl<E: FieldElement> TracePolyTable<E> {
     /// Adds the provided auxiliary segment polynomials to this polynomial table.
     pub fn add_aux_segment(
         &mut self,
-        aux_segment_polys: ColMatrix<E>,
+        aux_segment_poly: ColMatrix<E>,
```

Comment:
nit: I think this should still be plural as we can have multiple columns in a single aux segment.

Reply:
fixed